### PR TITLE
Jormun: Use pypy for jormungandr

### PR DESCRIPTION
**Pull request by xlqian**
fix some unit tests to prepare the use of pypy; some other improvements

This PR is a recovered version for: 

https://github.com/CanalTP/navitia/pull/1108

### CODE REVIEW

**Comment by xlqian** [comment]
Jenkins, test this please

File: source/jormungandr/jormungandr/stat_manager.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def fill_request(self, stat_request, call_result):
             stat_request.client = request.headers.getlist("X-Forwarded-For")[0]
         stat_request.path = request.path
 
-        stat_request.response_size = sys.getsizeof(call_result[0])
+        stat_request.response_size = 4242
```
Comment (francois-vincent) [inline source/jormungandr/jormungandr/stat_manager.py:212]: why is this necessary ?

Comment (xlqian) [inline source/jormungandr/jormungandr/stat_manager.py:212]: This line should be removed or commented. 

The intention of using `sys.getsizeof()` is to get the size of response in terms of 'length', but in fact this function returns the whole size of a python object in memory, the use may be wrong  and secondly this function is not yet implemented in pypy...

**Comment by antoine-de** [comment]
close because it's been too long, and the new  solution will be to change flask/marshall to lima
